File: app/src/main/java/com/pcsalt/example/customalertdialogdemo2/MyEditText.java
package com.pcsalt.example.customalertdialogdemo2;

import android.content.Context;
import android.view.ViewGroup.LayoutParams;
import android.view.inputmethod.EditorInfo;
import android.widget.LinearLayout;

import androidx.appcompat.widget.AppCompatEditText;

/**
 * Created by Navkrishna on 12 October, 2017
 */

public class MyEditText extends AppCompatEditText {

    public MyEditText(Context context) {
        super(context);
        this.setSingleLine();
        this.setImeOptions(EditorInfo.IME_ACTION_DONE);
        this.setImeActionLabel("Done", EditorInfo.IME_ACTION_DONE);
        this.setLayoutParams(new LinearLayout.LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.WRAP_CONTENT));
    }
}

File: app/src/main/java/com/pcsalt/example/customalertdialogdemo2/MyEditText.kt
package com.pcsalt.example.customalertdialogdemo2

import android.content.Context
import android.view.ViewGroup
import android.view.inputmethod.EditorInfo
import android.widget.LinearLayout
import androidx.appcompat.widget.AppCompatEditText

/**
 * Created by Navkrishna on 12 October, 2017
 */
class MyEditText(context: Context?) : AppCompatEditText(context!!) {
    init {
        this.setSingleLine()
        this.imeOptions = EditorInfo.IME_ACTION_DONE
        setImeActionLabel("Done", EditorInfo.IME_ACTION_DONE)
        this.layoutParams = LinearLayout.LayoutParams(
            ViewGroup.LayoutParams.MATCH_PARENT,
            ViewGroup.LayoutParams.WRAP_CONTENT
        )
    }
}